misc/master.sql

```diff
@@ -319,7 +321,10 @@ CREATE TABLE `storageUploadQueue` (
   `charset` varchar(25) DEFAULT NULL,
   `time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   PRIMARY KEY (`uploadKey`),
-  KEY `userID` (`userID`)
+  KEY `userID` (`userID`),
+  KEY `libraryID` (`libraryID`),
+  KEY `key` (`key`),
```

Comment:
Oh yeah, 'key' isn't queried anywhere. Just 'libraryID' need index, because it has foreign constraint, and 'hash' because we're querying by it.